federation/pkg/federation-controller/cluster/clustercontroller.go

```diff
@@ -66,10 +68,14 @@ func NewclusterController(federationClient federationclientset.Interface, cluste
 	cc.clusterStore.Store, cc.clusterController = framework.NewInformer(
 		&cache.ListWatch{
 			ListFunc: func(options api.ListOptions) (runtime.Object, error) {
-				return cc.federationClient.Federation().Clusters().List(options)
+				// TODO: remove this when Reflector takes an interface rather than a particular ListOptions as input parameter.
```

Comment:
I think one TODO for the function itself (that you have) should be enough. There you can mention all calls to this method should be removed after ...